packages/react-components/react-card/src/components/Card/useCard.ts

```diff
@@ -28,7 +28,6 @@ export const useCard_unstable = (props: CardProps, ref: React.Ref<HTMLElement>):
 
   return {
     appearance,
-    focusMode,
```

Comment:
not sure if this is ok cc @andrefcdias ?

Reply:
I'd say it's fine as `focusMode` will only be used inside the hook.